### PR TITLE
feat: support migration from Matic to Pol

### PR DESCRIPTION
**Pull request by myz1237**
https://lifi.atlassian.net/browse/LF-9658
- Change matic chain information
   - In terms of coins, I would like to keep Matic coins for other chains except matic chain, after the migration, matic may be still exchangable in some dex or place. Add Pol coins for all chains, including matic chain
   - Change wmatic token to wpol token symbol

### CODE REVIEW

File: src/coins/coins.ts [hunk 1]
```diff
@@ -168,6 +164,40 @@ export const basicCoins: BasicCoin[] = [
       },
     },
   },
+  // > WMATIC
+  {
```
Comment (myz1237) [inline src/coins/coins.ts:168]: Keep wmatic in the system

File: src/coins/coins.ts [hunk 2]
```diff
@@ -152,10 +152,6 @@ export const basicCoins: BasicCoin[] = [
         decimals: 18,
         name: 'Matic Token',
       },
-      [ChainId.POL]: {
```
Comment (myz1237) [inline src/coins/coins.ts:155]: keep matic in the system, and remove native matic

File: src/coins/coins.ts [hunk 3]
```diff
@@ -1646,11 +1662,11 @@ export const wrappedTokens: { [ChainId: string]: StaticToken } = {
...
     // https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270
     address: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270',
-    symbol: 'WMATIC',
+    symbol: 'WPOL',
     decimals: 18,
     chainId: ChainId.POL,
-    coinKey: 'WMATIC' as CoinKey,
-    name: 'WMATIC',
+    coinKey: CoinKey.WPOL,
+    name: 'WPOL',
```
Comment (myz1237) [inline src/coins/coins.ts:1669]: Wmatic will be migrated to wpol with the same address

**Review by melianessa — APPROVED** [review]
looks good to me :)

**Pull request merged**